dali/test/python/test_operator_crop_mirror_normalize.py

```diff
@@ -154,9 +154,9 @@ def crop_mirror_normalize_func(crop_y, crop_x, crop_h, crop_w, mirror_probabilit
         F, H, W, C = image.shape[0], image.shape[1], image.shape[2], image.shape[3]
     assert H >= crop_h and W >= crop_w
 
-    start_y = int(np.round(np.float32(crop_y) * np.float32(H - crop_h)))
+    start_y = int(np.float32(crop_y) * np.float32(H - crop_h) + 0.5)
```

Comment:
I think we should do
```suggestion
    start_y = int(np.float32(crop_y) * np.float32(H - crop_h) + np.float32(0.5))
```
I'll submit a patch